File: edabitPython/x_pronounce.py
def x_pronounce(sentence):
	strList = sentence.split(" ")
	sol = ""
	for word in strList:
		if len(word) == 1 and word == "x":
			sol += " ecks"
		elif word[0] == "x":
			sol += (" z"+ word[1:].replace("x", "cks"))
		else:
			sol += " "+word.replace("x", "cks")
	return sol.strip()
	

File: edabitJavascript/xPronounce.js
function xPronounce(sentence) {
  let arr = sentence.split(" ").map((word) => {
    if (word.length == 1 && word == "x") {
      return "ecks";
    } else if (word[0] == "x") {
      word = "z" + word.slice(1);
    }
    return word.replace(/[x]/g, "cks");
  });
  return arr.join(" ");
}
